projects/go-lib/src/lib/components/go-button/go-button.component.scss

```diff
@@ -113,6 +115,15 @@
   }
 }
 
+.go-button--secondary {
+  &:hover,
+  &:focus,
+  &:active {
+    border: 1px solid lighten($base-light-secondary, 10%);
+    box-shadow: 0 0 0 3px transparentize($base-light-tertiary, .75);
```

Comment:
Should this be one of the existing `form-show-...` variables, or perhaps a new one?

Reply:
@jaredami this is a very specific use case. We already have this variable:
```
$form-shadow-hover: 0 0 0 3px transparentize($base-light-tertiary, .5);
```
but on the button we needed the transparency to be 25% instead of 50%. Thus I'm not sure it's necessary to create another variable specific to this one spot.